conda/base/context.py

```diff
@@ -492,6 +492,10 @@ def execute_threads(self):
     def subdir(self):
         if self._subdir:
             return self._subdir
+        return self._native_subdir()
+
+    @lru_cache(maxsize=None)
+    def _native_subdir(self):
```

Comment:
Could we make this a `memoizedproperty` instead and remove the `_` prefix?

Reply:
[```
        "The `conda.auxlib.decorators.memoize` decorator is pending deprecation and will be "
        "removed in a future release. Please use `functools.lru_cache` instead.",
```](https://github.com/conda/conda/blob/main/conda/auxlib/decorators.py#L53)

There may not be a working/non-deprecated function with a prettier name

Reply:
Ah, I knew that, and forgot 😬 Ignore me!